metaparser/xcarchive.go

```diff
@@ -41,8 +48,7 @@ func (m *Parser) readXCArchiveDeploymentMeta(pth string) (Info, string, error) {
 	xcarchiveReader := artifacts.NewXCArchiveReader(reader)
 	isMacos := xcarchiveReader.IsMacOS()
 	if isMacos {
-		m.logger.Warnf("macOS archive deployment is not supported, skipping xcarchive")
-		return Info{}, "", nil // MacOS project is not supported, so won't be deployed.
+		return Info{}, "", MacOSProjectIsNotSupported // so won't be deployed.
```

Comment:
This comment is not needed, it is up to the client what will happen with such an artifact